vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -97,15 +106,17 @@ void Notifications::CloseAll() const {
   DCHECK(is_initialized_);
 
   for (const auto& notification : notifications_) {
-    auto id = notification.first;
-    ads_client_->CloseNotification(id);
+    ads_client_->CloseNotification(notification.id);
   }
+  //the diff with RemoveAll: it doesn't save state
 }
 
 bool Notifications::Exists(const std::string& id) const {
   DCHECK(is_initialized_);
 
-  if (notifications_.find(id) == notifications_.end()) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
+      [&id](const auto & v){return v.id == id;});
```

Comment:
Please change v to notification

For. Insistence can you please use same formatting/style as rest if ads/confirmations for lambdas, thanks

Reply:
done